### PR TITLE
Attempt to prevent Travis timeouts

### PR DESCRIPTION
**Pull request by kidder**
## Proposed changes

Build more test libraries during the first stage of the Travis build

### Types of changes:

- [ ] Bugfix
- [ ] New feature

### Component:

- [ ] Code
- [ ] Documentation
- [ ] Build system
- [x] Continuous integration

### Code review checklist

- [ ] The PR passes all checks, including unit tests, `clang-tidy` and `IWYU`.
  For instructions on how to perform the CI checks locally refer to the [Dev
  guide on the Travis CI](https://spectre-code.org/travis_guide.html).
- [ ] The code is documented and the documentation renders correctly. Run
  `make doc` to generate the documentation locally into `BUILD_DIR/docs/html`.
  Then open `index.html`.
- [ ] The code follows the stylistic and code quality guidelines listed in the
  [code review guide](https://spectre-code.org/code_review_guide.html).

### CODE REVIEW

File: .travis/BuildLinux.sh [hunk 1]
```diff
@@ -57,15 +57,15 @@ if [ -z "${RUN_CLANG_TIDY}" ] \
     && [ -z "${RUN_IWYU}" ] \
     && [ -z "${BUILD_DOC}" ]; then
     if [[ ${TRAVIS_BUILD_STAGE_NAME} = "Build libraries" ]]; then
-        make libs test-libs-stage1 -j2
+        time make libs test-libs-stage1 -j2
```
Comment (nilsdeppe) [inline .travis/BuildLinux.sh:60]: Do you want to keep the timing records? I personally have no objection to them :)

**Review by nilsdeppe — APPROVED** [review]

File: .travis/BuildLinux.sh (hunk 1 above)
Comment (kidder) [reply .travis/BuildLinux.sh:60]: yeah, I don't see how they hurt...

**Comment by kidder** [comment]
@wthrowe please look at (and merge if you approve)

**Review by wthrowe — APPROVED** [review]
This will hopefully keep things going for the moment, but it doesn't look like it will last long.  One of the builds for this PR was within a minute of timing out.

**Pull request merged**